static/styles/portico/portico.css

```diff
@@ -433,6 +433,61 @@ img.screenshot {
     #portico-area {
         margin-left: 1.5em;
     }
+
+    /* To reveal the whole page for printing we should show the content of the body,
+    since printing dialog can't scroll content inside nested containers (simplebar) as a user. */
+    body.noscroll {
+        overflow: auto;
+        position: static;
+        width: 100%;
+    }
```

Comment:
What's the thinking around scoping this to `body`?  I believe that'll apply to all the logged-out pages; did you test or reason about /hello, /features, /integrations, /login and other pages that aren't part of the /help/ framework?